app/views/salaries/edit.html.haml

```diff
@@ -0,0 +1,3 @@
+%h2= "Edit Salary for #{@salary.employee_first_name} #{@salary.employee_last_name}"
```

Comment:
This interpolation via using HAML could be simplified to remove the `=` and `" "`, so something like:
```
%h2
    Edit Salary for #{@salary.employee_first_name} #{@salary.employee_last_name}
```
would produce the same outcome